### PR TITLE
Minor map changes

### PR DESCRIPTION
**Pull request by SomeCabbage**
# Description
Added telecom servers to the AI core room
Fixed security's disposal chute not working
Added more GPSs to salvage
Reoriented maints bar sign because it was upside-down
Changed docking ports at arrivals to be the new arrivals dock type (not locked)
Finally fixed HoP's locker and it's now the filled one and also added durathread to the catalog

Works in release build testing and loads correctly.

<details><summary><h1>Media</h1></summary>
<p>

AI core.
map1

Security disposals.
map2

Salvage GPSs
map3

Arrivals docklocks.
map4

</p>
</details> 

# Changelog

:cl: SomeCabbage
- tweak: Several minor Parkstation map tweaks. Reoriented the maintenance bar sign. Arrivals docking airlocks are now the new type of arrival airlock meant for the arrival shuttle. The Security disposal chute is now hooked up. AI core now has telecom servers. Salvage now has 3 pre-placed GPSs. HoP locker is better now and contains cool stuff.

### CODE REVIEW

**Comment by SomeCabbage** [comment]
reformatted the PR to look nicer and also added one thing

**Comment by DEATHB4DEFEAT** [comment]
I feel like only one CL is necessary

**Comment by SomeCabbage** [comment]
there

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml
Comment (Pspritechologist) [inline Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:1]: Map updates should generally only be map updates.

**Review by Pspritechologist — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Lockers/heads.yml
Comment (SomeCabbage) [reply Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:1]: dammit, I'll fix that

Comment (SomeCabbage) [reply Resources/Prototypes/Catalog/Fills/Lockers/heads.yml:1]: reverted now

**Review by Pspritechologist — APPROVED** [review]

**Pull request merged**